components/utilities/text-utilities/text-utilities.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700;800&display=swap");
+* {
+    font-family: "Open Sans", sans-serif;
+}
```

Comment:
Why do this here again if mentioned in :root?

Reply:
things were not getting applied like font-weight. By doing this it was working.